install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+= Direct access to storage using two methods
```

Comment:
The title is a bit confusing to me. It's not clear what "direct access" means, though I understand it from your examples. Maybe:
"Ensuring a Pod has Access to the Target Persistent Storage"  -- though that's not great...